package.json

```diff
@@ -35,6 +35,13 @@
     "typescript": "4.4.2"
   },
   "dependencies": {
-    "react": "17.0.2"
+    "@patternfly/patternfly": "4.132.2",
+    "@patternfly/react-core": "4.152.4",
+    "@patternfly/react-table": "4.29.58",
+    "@patternfly/react-icons": "4.11.14",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-router-dom": "5.2.1",
+    "tslib": "2.3.1"
```

Comment:
Shouldn't all these be peer dependencies rather than direct dependencies?

Reply:
I'll move them thanks for the comment.